postgresql/scale-postgresql-server/scale-postgresql-server.sh

```diff
@@ -14,7 +14,7 @@ az postgres server create \
 --location westus \
```

Comment:
__Required, style__: Indent 4 spaces after line continution, needs to be done throughout. See https://review.docs.microsoft.com/en-us/help/contribute/conventions-azure-cli?branch=master

Reply:
L12:

__Required__: The name of the database is required to be a unique identifier but isn't. See https://review.docs.microsoft.com/en-us/help/contribute/conventions-azure-cli?branch=master#avoid-naming-conflicts